charm/requirements.txt

```diff
@@ -0,0 +1 @@
+paas-app-charmer==1.*
```

Comment:
should we pin this to a specific version? Is the paas-app-charmer stable enough to have the wildcard?